noxfile.py

```diff
@@ -44,6 +44,8 @@ def test(session: nox.Session) -> None:
 
 @nox.session(name="test-rust", venv_backend="none")
 def test_rust(session: nox.Session):
+    _run_cargo_package_metadata_test(session, packages=["pyo3", "pyo3-ffi"])
```

Comment:
Will `pyo3` and `pyo3-ffi` always support exactly the same versions? I guess so. Should we ensure their metadata is always in sync? Or should we have it only on `pyo3-ffi`? 🤔

Reply:
> Or should we have it only on `pyo3-ffi`?

Good point, I think have it only on a leaf dependency makes sense!